README.md

```diff
@@ -19,6 +19,27 @@ $ npm install ffmpeg-static
 
 *Note:* During installation, it will download the appropriate `ffmpeg` binary from the [`b4.4.1` GitHub release](https://github.com/eugeneware/ffmpeg-static/releases/tag/b4.4.1). Use and distribution of the binary releases of FFmpeg are covered by their respective license.
 
+### Custom binaries url
+
+To use a mirror of the ffmpeg-static binaries use npm config property `ffmpeg_binaries_url`.
+Default is `https://github.com/eugeneware/ffmpeg-static/releases/download`.
+
+```shell
+npm install ffmpeg-static --ffmpeg_binaries_url=https://cdn.npmmirror.com/binaries/ffmpeg-static
+```
+
+Or add property into your [`.npmrc`](https://docs.npmjs.com/files/npmrc) file.
+
+```
+ffmpeg_binaries_url=https://cdn.npmmirror.com/binaries/ffmpeg-static
+```
+
+Another option is to use PATH variable `FFMPEG_BINARIES_URL`.
+
+```shell
+FFMPEG_BINARIES_URL=https://cdn.npmmirror.com/binaries/ffmpeg-static npm install ffmpeg-static
```

Comment:
Just setting the variable won't export it, so it won't be accessible to `ffmpeg-static`'s `node install.js` (which is a child process of `npm install`).

```suggestion
env export FFMPEG_BINARIES_URL=https://cdn.npmmirror.com/binaries/ffmpeg-static npm install ffmpeg-static
```